tests/test_commands.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+
+def test_inquire(mapdl):
+
+    # Testing basic functions (First block: Functions)
+    assert 'apdl' in mapdl.inquire('apdl').lower()
+
+    # **Returning the Value of an Environment Variable to a Parameter**
+    env = list(os.environ.keys())[0]
+    if os.name == 'nt':
+        env_value = os.getenv(env).split(';')[0]
+    elif os.name == 'posix':
+        env_value = os.getenv(env).split(':')[0]
+    else:
+        raise Exception('Not supported OS.')
+
+    env_ = mapdl.inquire('ENV', env, 0)
+    assert env_ == env_value
+
+    # **Returning the Value of a Title to a Parameter**
```

Comment:
Avoid bold in comments. This isn't rst and it's just developer notes.

```suggestion
    # returning the value of a title to a parameter
```